src/App/Pages/Vault/AddEditPage.xaml.cs

```diff
@@ -30,7 +30,9 @@ public partial class AddEditPage : BaseContentPage
             string name = null,
             string uri = null,
             bool fromAutofill = false,
-            AppOptions appOptions = null)
+            AppOptions appOptions = null,
+            bool cloneMode = false,
+            ViewPage viewPage = null)
```

Comment:
@kspearrin Is this how you envisioned the instance of ViewPaged being passed into the constructor?

Reply:
Yes, looks good